Lesson5/README.md

```diff
@@ -0,0 +1,89 @@
+# Compile and run Azure gateway SDK module and sample application
+This sample accompanies [Compile and run Azure gateway SDK module and sample application](#) lesson. You will use scripts to compile the Azure gateway SDK's module and sample application yourself.
+
+## Prerequisites
+See associate lesson for more information.
+
+## Repository information
+
+```txt
+.
+|- module\                    
+|    |- hello_world\           
+|        |- src\                 // Azure gateway SDK hello_world module source code
+|        |   |- hello_world.c
+|        |   |- hello_world.h
+|        |- build.sh             // build script for hello_world module
+|- sample\
+|    |- hello_world\
+|        |- src\                    // Azure gateway SDK hello_world sample application source code
+|        |   |- hello_world.json
+|        |   |- main.c
+|        |- build.sh                // build script for hello_world sample application
+|- config.json
+|- gulpfile.js
+```
+
+## Compile and run hello_world sample
+
+These steps will guide you compile the Azure gateway SDK's hello_world sample application on your Intel NUC. This sample application creates a gateway from hello_world.json file and use two modules in Azure IoT gateway SDK to log a "hello world" message to a file every 5 seconds.

```

Comment:
These steps will guide you through compilation of Azure gateway SDK's hello_world sample application on your Intel NUC. This sample application creates a gateway from hello_world.json file which includes use two modules. The gateway logs "hello world" message every 5 seconds.